packages/discord.js/typings/index.d.ts

```diff
@@ -1154,6 +1154,7 @@ export class Guild extends AnonymousGuild {
   public setVerificationLevel(verificationLevel: GuildVerificationLevel | null, reason?: string): Promise<Guild>;
   public setPremiumProgressBarEnabled(enabled?: boolean, reason?: string): Promise<Guild>;
   public setWidgetSettings(settings: GuildWidgetSettingsData, reason?: string): Promise<Guild>;
+  public setMFALevel(options: GuildMFALevelOptions): Promise<Guild>;
```

Comment:
```suggestion
  public setMFALevel(level: GuildMFALevel, reason?: string): Promise<Guild>;
```

Also get rid of the `GuildMFALevelOptions` interface.